README.md

```diff
@@ -8,32 +8,27 @@ I want to install Mattermost on my Raspberry Pi for use on my network.
 
-The builds are updated regularly, so the most recent version of Mattermost (5.21) is available. There are installers for quite a few flavors of Linux. The version I used was **Linux-arm-tar.gz** as I'm running Raspbian 9 (stretch) on my Raspberry Pi 3 Model B which has an ARM Cortex-A53 processor. You can check this by running *uname --kernel-name --kernel-release --machine*. If your Pi is out the box, the results should be something like **Linux 4.14.34-v7+ arm71**. If you've installed a different flavor of Linux, it'll be different so choose the appropriate install file.  
+The builds are updated regularly, so the most recent version of Mattermost (5.21) is available. There are installers for quite a few flavors of Linux. The version I used was **Linux-arm-tar.gz** as I'm running Raspbian 10 (buster) on my Raspberry Pi 4 Model B which has an ARM. You can check this by running *uname --kernel-name --kernel-release --machine*. If your Pi is out the box, the results should be something like **Linux 4.19.97-v7l+ armv7l**. If you've installed a different flavor of Linux, it'll be different so choose the appropriate install file.  
 
-Setting up MySQL
-----------------
-Before you install Mattermost, you need to set up MySQL. The full instructions are on the [Installing Mattermost on Debian Stretch](https://docs.mattermost.com/install/install-debian.html/) page and cover PostgreSQL and MySQL. I use MySQL, so these are the steps I followed: 
+Setting up MariaDB
+------------------
+Before you install Mattermost, you need to set up MariaDB. The full instructions are on the [Installing Mattermost on Debian Stretch](https://docs.mattermost.com/install/install-debian.html/) page and cover PostgreSQL and MySQL. I use MariaDB (a drop-in replacement of MySQL), so these are the steps I followed: 
 
 1. Log into the server that will host the database, and open a terminal window.
 
-2. Download the MySQL repository package 
+2.  your local package list
 ```bash
-wget https://dev.mysql.com/get/mysql-apt-config_0.8.6-1_all.deb
+sudo apt update
 ```
 
-3. Install the repository 
+3. Install MariaDB
 ```bash
-sudo dpkg -i mysql-apt-config*
+sudo apt install mariadb-server
 ```
 
-4. Update your local package list
+4. Become the root user
 ```bash
-sudo apt-get update
-```
-
-5. Add the MySQL repo MySQL. During the install, you’ll be prompted to create a password for the MySQL root user. Make a note of the password because you’ll need it in the next step.
-```bash
-sudo apt-get install mysql-server
+sudo su -
 ```
 
 6. Log in to MySQL as root, using the root password that you created when installing MySQL.
```

Comment:
Based on the changes above, it looks like this line should also be removed?

Reply:
I just tried to run the "sudo mysql -u root -p" command, just as the pi user, not root.  And it works!  So my "sudo su -" is superfluous (all of step 4).  It can be removed.  Also the exit from the root privileges (step 11).  Somehow you never do set a password on the root user *of MariaDB*, and having plain-old root privileges *within Raspbian* suffices, it seems.  Somebody please correct me, if I'm wrong here.